cogs/covidcog.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord.ext import commands
+import requests
+
+
+def embedCreator(data):
+    state = data["state"]
+    embedTitle = ":world_map: Covid Stats in " + state + " :world_map:"
+    firstField = {
+        "name": "Total " + state + " Cases",
+        "value": "Total Cases: {:,}".format(data["cases"]),
+        "inline": True
+    }
+    secondField = {
+        "name": "Total " + state + " Deaths",
+        "value": "Total Deaths: {:,}".format(data["deaths"]),
+        "inline": True
+    }
+    thirdField = {
+        "name": "Total " + state + " Tests",
+        "value": "Total Tests: {:,}".format(data["tests"]),
+        "inline": True
+    }
+    embed = discord.Embed(title=embedTitle, color=0x8D0000)
+    embed.description = "Statistics for Covid-19 in " + state + "\n"
+    embed.add_field(
+        name=firstField["name"], value=firstField["value"], inline=firstField["inline"])
+    embed.add_field(
+        name=secondField["name"], value=secondField["value"], inline=secondField["inline"])
+    embed.add_field(
+        name=thirdField["name"], value=thirdField["value"], inline=thirdField["inline"])
+    embed.set_footer(text="Data from corona.lmao.ninja")
+
+    return embed
+
+
+class CovidCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def covid(self, ctx):
+        r = requests.get("https://corona.lmao.ninja/v2/states")
+        apidata = r.json()
+        state = ctx.message.content[7:].title()
+        returningData = {}
+        for stateData in apidata:
```

Comment:
You can replace this with a one-liner:
```python
# Unpack the head of a list of state_data points whose state corresponds to the given state
# name, and discard the tail
returning_data, *_ = [state_data for state_data in apidata if state_data["state"] == state]
```

However, I am starting to feel like we could make some minor changes to variable names. Python uses `snake_case` for variable and function names, and I believe `state_name` would be a better name for the `state` variable/parameter.